scripts/create_service.sh

```diff
@@ -26,8 +26,13 @@ fi
 
 case "${language}" in
     go)
-        build_script="cd \"traffic-director-grpc-examples/go/${service_type}_server\"
-sudo apt-get install -y golang
+        build_script="sudo apt-get install -y wget
+wget https://dl.google.com/go/go1.16.5.linux-amd64.tar.gz
+sudo tar -C /usr/local -xvf go1.16.5.linux-amd64.tar.gz
+sudo cp /usr/local/go/bin/go /usr/bin/go
+cd \"traffic-director-grpc-examples/go/${service_type}_server\"
+go build .
+go get google.golang.org/grpc
```

Comment:
Why is this necessary?
The `go build .` above should download the dependencies based on `go.mod`.

Reply:
Without this, we would have to update this `go.mod` whenever we have a new gRPC version. Is that what we want?

Reply:
> update this go.mod whenever we have a new gRPC version

I think that is the right thing to do.